foreman/foreman.spec

```diff
@@ -368,6 +370,8 @@ Requires: %{?scl_prefix_ror}rubygem(uglifier) >= 1.0.3
 Requires: %{?scl_prefix_ror}rubygem(execjs) >= 1.4.0
 Requires: %{?scl_prefix_ror}rubygem(jquery-rails) >= 3.1
 Requires: %{?scl_prefix_ror}rubygem(jquery-rails) < 4.0
+Requires: %{?scl_prefix_ror}rubygem(sprockets-rails) >= 2.3.3
```

Comment:
ditto above.

Also http://projects.theforeman.org/issues/13074 should be fixed as having this dependency only in foreman-assets could lead to the wrong version being installed via the main foreman package.